manifest/manifest.go

```diff
@@ -279,6 +293,58 @@ func stringVal(yamlMap map[string]interface{}, key string, errs *[]error) *strin
 	return &result
 }
 
+func bytesVal(yamlMap map[string]interface{}, key string, errs *[]error) *int64 {
+	yamlVal := yamlMap[key]
+	if yamlVal == nil {
+		return nil
+	}
+
+	stringVal := coerceToString(yamlVal)
+	value, err := formatters.ToMegabytes(stringVal)
+	if err != nil {
+		*errs = append(*errs, fmt.Errorf("Invalid value for '{{.PropertyName}}': {{.StringVal}}\n{{.Error}}",
+			map[string]interface{}{
+				"PropertyName": key,
+				"Error":        err.Error(),
+				"StringVal":    stringVal,
+			}))
+		return nil
+	}
+	return &value
+}
+
+func intVal(yamlMap map[string]interface{}, key string, errs *[]error) *int {
```

Comment:
There is now a test for Applications that makes sure that these values are parsed